build/operations/README.md

```diff
@@ -0,0 +1,171 @@
+
+The FAST Front Door will perform caching and dynamic content acceleration. Front Door can cache your static content and directly return cached assets to your customers without a trip to your backend further reducing latency.
+
+#### Limitations
+Several limitations exist between Azure Front Door and Azure Active Directory (AAD).
+
+* Front Door does not support using Response rewriting
+* MIME Types: There are certain limitations on fonts, images, an data files.
+
+For additional limitations visit [details](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#azure-front-door-service-limits) https://docs.microsoft.com/en-us/azure/frontdoor/front-door-caching
+
+_Note_ that Application Gateway has this capability. A new feature request has been sent to Azure Front Door.
+
+**Issue**
+We use Front Door to route traffic between two regions for web traffic across multiple subdomains.  We have subdomains for https://stage.www.fast.design, for example, that we want to protect behind AAD.  This works great when configuring AAD Reply URLs to match the backend server name/url.  In this example, https://www-west-app.azurewebsites.net//.auth/login/aad/callback.  However, we want users to return back to the original URL of https://stage.www.fast.design not the backend URL.  
+
+How can this be achieved through the Azure Portal, not web app code?
+
+**Cause**
+-Issue happens because of URL string within one of the parameters of the HTTP redirect pointing to AAD, which then uses such parameter for redirecting the user accordingly.
+-Such URL was the one configured on the WebApp.
+
+**Resolution**
+-We managed to configure custom domain on both the WebApp and FrontDoor, so they both listen on same hostname. Such Hostname or FQDN, would CNAME to FrontDoor's domain.
+-This custom domain was now used on the URL string within the AAD redirect so would work as expected. Nevertheless, since such AAD module responds with 302 to every unauthenticated user-agent, the FrontDoor probes would mark backend as unhealthy:
+
+https://docs.microsoft.com/en-us/azure/frontdoor/front-door-health-probes
+
+-When having multiple backends, it would round-robin requests, making it unusable.
+-Usually a dummy path is setup in application for the 200 OK probes, but verified with AppServices it cannot be made with the pre-built AAD authentication module. To customize AAD, it's then suggested to implement it a code.
+
+-This scenario doesn't happen with AppGW, since it can just re-write the URL parameter and also allow non-200 OK health responses.
```

Comment:
Looks like formatting was a little off here.

```suggestion
- Issue happens because of URL string within one of the parameters of the HTTP redirect pointing to AAD, which then uses such parameter for redirecting the user accordingly.
- Such URL was the one configured on the WebApp.

**Resolution**
- We managed to configure custom domain on both the WebApp and FrontDoor, so they both listen on same hostname. Such Hostname or FQDN, would CNAME to FrontDoor's domain.
- This custom domain was now used on the URL string within the AAD redirect so would work as expected. Nevertheless, since such AAD module responds with 302 to every unauthenticated user-agent, the FrontDoor probes would mark backend as unhealthy:

https://docs.microsoft.com/en-us/azure/frontdoor/front-door-health-probes

- When having multiple backends, it would round-robin requests, making it unusable.
- Usually a dummy path is setup in application for the 200 OK probes, but verified with AppServices it cannot be made with the pre-built AAD authentication module. To customize AAD, it's then suggested to implement it a code.

- This scenario doesn't happen with AppGW, since it can just re-write the URL parameter and also allow non-200 OK health responses.
```